turingschool/savile
# NOT MERGE READY: Style doc site

‼️Updated after Tanner's comments were made
#### What does this PR do?

- Styles header to match Terminal and fit a bit better with that design
- Adjust footer responsiveness issue
- Add favicon
- Adjust left-side nav bar, some small styling things as well as add sub-nav items
- Add in rouge, apply to code snippets

#### Where should the reviewer start?

- Screenshots/run locally to look at new styles. Header, nav, tables, code snippets.

#### How should this be manually tested?

- Click around, see how you like interacting with the nav bar. Does it bother you that when you click a menu item, if a different menu item was previously selected, it's sub-items disappear? (Try clicking tokens, then elements)

#### FYI/Context

- To handle the nav bar subitem situation, I decided to use that `if` in liquid rather than CSS because the CSS solution got hairy and required way more code that felt less straightforward.
 
#### What are the relevant tickets?

https://3.basecamp.com/3494409/buckets/19192671/todolists/3628883354

#### Screenshots (if appropriate)

This screenshot shows new header, side-nav bar with a sub-item available under elements, and the rouge integrated:

<img width="1552" alt="Screen Shot 2021-04-07 at 7 09 28 AM" src="https://user-images.githubusercontent.com/25447342/113871552-408c4d00-9770-11eb-88ff-827f647542dc.png">

This screenshot shows updated tables that are more consistent in widths of cells:

<img width="1552" alt="Screen Shot 2021-04-07 at 7 10 23 AM" src="https://user-images.githubusercontent.com/25447342/113871648-5ac62b00-9770-11eb-8e2e-39110078930b.png">

This one shows three table on tokens page, all with the same widths:

<img width="1552" alt="Screen Shot 2021-04-07 at 7 11 04 AM" src="https://user-images.githubusercontent.com/25447342/113871754-74677280-9770-11eb-959c-ddac1c25bedf.png">

#### Any other deploy steps?

No
